app/src/main/java/org/simple/clinic/cvdrisk/CVDRiskCalculator.kt

```diff
@@ -0,0 +1,71 @@
+package org.simple.clinic.cvdrisk
+
+import org.simple.clinic.patient.Answer
+import org.simple.clinic.patient.Gender
+
+object CVDRiskCalculator {
+
+  fun calculateCvdRisk(
+      cvdRiskData: CVDRiskCalculationSheet?,
+      gender: Gender,
+      age: Int,
+      sbp: Int,
+      isSmoker: Answer = Answer.Unanswered,
+      bmi: Double? = null
```

Comment:
I think a `Float` precision should be enough for BMI

Reply:
Done